packages/battery_plus/battery_plus/pubspec.yaml

```diff
@@ -34,7 +34,7 @@ dependencies:
   battery_plus_platform_interface: ^2.0.1
   meta: ^1.8.0
   upower: ^0.7.0
-  web: ^0.5.0
```

Comment:
Are we doing the range ">=0.5.1 <2.0.0" like the packages repo did for these?

https://github.com/flutter/packages/pull/7202

Reply:
In this case, switching to `^1.0.0` is preferable since we want to use the newly available APIs changes in the web package, i.e. `getBattery()`

Reply:
We kept the custom extensions in those packages in order to support both 0.5.0 and 1.0.0

Reply:
:thinking: I thought we agreed to remove the custom extensions since 1.0.0 already provide those types.

Personally, I am more keen to use 1.0.0 directly and remove the extensions, and only keep the range in cases where no extensions are involved (e.g. with other plugins like `device_info_plus`).

Reply:
My vote is for going ^1.0.0 here and in the share_plus.